images-list

```diff
@@ -46,3 +46,6 @@ directxman12/k8s-prometheus-adapter-amd64 rancher/directxman12-k8s-prometheus-ad
 bats/bats rancher/bats-bats v1.1.0
 curlimages/curl rancher/curlimages-curl 7.70.0
 busybox rancher/busybox 1.31.1
+coredns/coredns rancher/coredns-coredns 1.6.9
```

Comment:
This version already exists in Dockerhub and is multi-arch: 

https://hub.docker.com/repository/docker/rancher/coredns-coredns/tags?page=1

Reply:
Yes, but it was pushed manually by @cjellick. Should be automated.